lib/DataSift/ApiClient.php

```diff
@@ -67,7 +67,12 @@ static public function call(
         $ssl = $user->useSSL();
 
         // Build the full endpoint URL
-        $url = 'http'.($ssl ? 's' : '').'://'.$user->getApiUrl(). $user->getApiVersion() . '/'. $endPoint;
+        if ($method == 'ingest') {
```

Comment:
I'm not overly keen on this, should this not work more like how the streaming_url model works? Should the ingestion of ODP be done with a specific endpoint related. I'm not sure hard coding exceptions like this is the right way to be doing it.